app/routes/wallet/dids.py

```diff
@@ -19,13 +19,38 @@
 router = APIRouter(prefix="/v1/wallet/dids", tags=["wallet"])
 
 
-@router.post("", response_model=DID)
+@router.post("", response_model=DID, summary="Create Local DID")
 async def create_did(
     did_create: Optional[DIDCreate] = None,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
-):
-    """Create Local DID."""
-    logger.debug("POST request received: Create DID")
+) -> DID:
+    """
+    Create Local DID
+    ---
+
+    This endpoint allows you to create a new DID in the wallet.
+    The `method` parameter is optional and can be set to 'key',
+    'sov', `web`, `did:peer:2` or `did:peer:4`.
+
+    The options field is deprecated it has been flattened and the `did` and
+    `key_type` fields are now top level fields. The `options` field will still
+    take precedence over the top level fields if it is present.
+
+    Request body:
+    ---
+        DIDCreate Optional:
+            method: [str]
+            options: [DIDCreateOptions]
+            seed: [str]
+            key_type: [str]
+            did: [str]
+
+    Response:
+    ---
+        Returns the created DID object.
+
+    """
+    logger.debug("POST request received: Create DID {}", did_create)
```

Comment:
I think if we want to log the request body, we should use a bound_logger, so the following logger.debug lines share the same context.
So it'll be bound_logger = logger.bind(body = did_create), and update to use bound_logger in this method. Something like that

Reply:
I'm gonna remove this. Was using it for some debugging